Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -343,6 +343,10 @@ define([
         this.batchTable.setAllColor(color);
     };
 
+    Batched3DModel3DTileContent.prototype.applyWireframe = function(enabled) {
+        this._model.debugWireframe = enabled;
+    };
```

Comment:
Instead of adding this function to every content type, it would be shorter to model `debugWireframe` after how `shadows` are set in `Instanced3DModel3DTileContent` and `Batched3DModel3DTileContent`.

It looks like you modeled it after `applyDebugSettings`. The reason that one is its own function is because it can be a quite heavy operation (`this.batchTable.setAllColor(color);`) that we wouldn't want to call every frame. For the case of `shadows` and `debugWireframe` though there is hardly any impact to do the simple updates every frame.